Gemfile

```diff
@@ -68,6 +68,7 @@ gem 'blacklight', '~> 7.18'
 gem 'blacklight-hierarchy', '~> 5.1'
 gem 'dor-services-client', '~> 7.0'
 gem 'dor-workflow-client', '~> 3.19'
+gem 'druid-tools'
```

Comment:
note that this gem is already included in argo because it is a dependency of the preservation-client gem, but this ensures we will have it even if that dependency goes away at some point